paper.md

```diff
@@ -168,27 +168,26 @@ In order to maximize its utility and target audience, DataLad is available for a
 Last but not least, in the absence of standardized data *packages*, there is no uniform way to declare actionable data dependencies and derivative relationships between inputs and outputs of a computation.
 DataLad aims to solve these issues by providing streamlined, transparent management of code, data, computing environments, and their relationship.
 It provides targeted interfaces and interoperability adapters to established scientific and commercial tools and services to set up unobstructed, unified access to all elements of scientific projects.
-This unique set of features enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that affords automatic re-execution.
-To this end, it builds up on and extends two established tools for version control and transport logistics, Git and git-annex.
+This unique set of features enables workflows that are particularly suited for reproducible science, such as actionable process provenance capture for arbitrary command execution that affords automatic re-execution.<!-- km: I think this use of "afford", while correct, is likely to throw some readers off -->
+To this end, it builds on and extends two established tools for version control and transport logistics, Git and git-annex.
 
 ## Why Git and git-annex?
 
 Git is the most popular version control system for software development[^1].
 It is a distributed content management system, specifically tuned towards managing and collaborating on text files, and excels at making all committed content reliably and efficiently available to all clones of a repository.
 At the same time, Git is not designed to efficiently handle large (e.g., over a gigabyte) or binary files [see, e.g., @opensource:git-binary].
 This makes it hard or impossible to use Git directly for distributed data storage with tailored access to individual files.
-Git-annex takes advantage of Git's ability to efficiently manage textual information to overcome this limitation.
+Git-annex takes advantage of Git's ability to efficiently manage textual information to overcome this limitation.<!-- km: I think this sentence could be dropped. -->
 File content handled by git-annex is placed into a managed repository annex, instead of being committing to Git directly.
-Rather than the actual file content, git-annex only commits a compact reference that enables identification and association of a file name with the content.
-Those file content references are typically based on a checksum of the content.
-Using these identifiers, git-annex tracks content availability across all repository clones (local or remote), or external resources such as URLs pointing to individual files on the web.
+Instead git-annex commits a compact reference, typically derived from the checksum of a file's content, that enables identification and association of a file name with the content.
```

Comment:
```suggestion
Instead, git-annex commits a compact reference, typically derived from the checksum of a file's content, that enables identification and association of a file name with the content.
```

Reply:
Apart from the missing comma, there is a doubling of "instead" with the previous sentence, maybe the previous sentence should drop the "instead" and use something like "…, which avoids committing the file content directly to Git."

Reply:
> here is a doubling of "instead" with the previous sentence, maybe the previous sentence should drop the "instead" and use something like "…, which avoids committing the file content directly to Git."

True, I didn't notice that.  I'll go with your suggestion.  Thanks.